packages/cli/src/commands/init.ts

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
chore: Could you format this file? Running prettier would fix some differences between this file and the rest of the project (example: double quotes-> single quotes, use of ";" etc)

Reply:
@lariciamota done. Thanks!